src/applets/tray/carbontray/child.c

```diff
@@ -18,22 +18,22 @@
 
 // static method header
 
-static void carbon_child_init(CarbonChild*);
-static void carbon_child_get_preferred_size(GtkWidget*, int*, int*);
-static bool set_wmclass(CarbonChild*, Display*);
-
+static void carbon_child_init(CarbonChild* child);
+static void carbon_child_get_preferred_size(GtkWidget* parent, int* minimumSize, int* naturalSize);
+static bool set_wmclass(CarbonChild* self, GdkDisplay* display, Display* xdisplay);
+static void synchronized_display_error_trap_push(GdkDisplay* display);
+static bool synchronized_display_error_trap_pop(GdkDisplay* display);
 
 
 // define our type with the macro
 
 G_DEFINE_TYPE(CarbonChild, carbon_child, GTK_TYPE_SOCKET)
 
 
-
 // public method implementations
 
 CarbonChild* carbon_child_new(int size, bool shouldComposite, GdkScreen* screen, Window iconWindow) {
-	if (GDK_IS_SCREEN(screen) == FALSE) {
+	if (GDK_IS_SCREEN(screen) == false) {
```

Comment:
Why not just `!GDK_IS_SCREEN(screen)` ?

Reply:
I'm not going to respond to all of these, but I guess I just assumed that the ! operator wouldn't work on ints. Now I realize that it has to, because C doesn't have bools.